bin/package_builder_push

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+###########################################################################
+# Creates Docker image and uploads it to DockerHub https://hub.docker.com/r/mysteriumnetwork/mysterium-node-builder/
+#
+# To create a release, simply execute:
+#> bin/package_builder_push version [docker_tag]
+
+# Usage example:
+#> bin/server_builder_docker 0.0.1
+#> bin/server_builder_docker 0.0.1 latest
+
+VERSION=$1
+if [ -z "$VERSION" ]; then
+    printf "\e[0;31m%s\e[0m\n" "Missing version!"
+    exit 1
+fi
+
+DOCKER_IMAGE="mysteriumnetwork/mysterium-node-builder"
+DOCKER_TAG=$2
+if [ -z "$DOCKER_TAG" ]; then
+    DOCKER_TAG=$VERSION
+fi
+
+printf "Building Docker image '$DOCKER_IMAGE:$DOCKER_TAG' ..\n" \
+    && docker build \
+        --file bin/util_docker/builder/Dockerfile \
+        --build-arg PACKAGE_VERSION=$VERSION \
+        --tag $DOCKER_IMAGE:$DOCKER_TAG \
+        . \
+    && docker push $DOCKER_IMAGE:$DOCKER_TAG
```

Comment:
Could be align this script in `server_package_docker` 
- `builder_package` - builds just local image
- `builder_package latest` - builds && pushes to Docker hub